CONNECTIVITY03 is to hard on single AS

The specification of [CONNECTIVITY03] gives ERROR if all name server IPv4 or IPv6 addresses are announce by one ASN. That is too hard. The implementation at the time of writing is not up-to-date with the specification and only returns WARNING for the same situation. We have received many reactions on WARNING.

I suggest that we modify this immediately. See what [ERROR] and [WARNING] should mean according to "Severity Level Definitions" (in develop branch).

From [CONNECTIVITY03]:

Message            |Default severity level
:------------------|:------------
IPV4_ONE_ASN       |ERROR        
IPV4_SAME_ASN      |NOTICE       
IPV4_DIFFERENT_ASN |INFO         
IPV6_ONE_ASN       |ERROR        
IPV6_SAME_ASN      |NOTICE       
IPV6_DIFFERENT_ASN |INFO         

[CONNECTIVITY03]: https://github.com/zonemaster/zonemaster/blob/master/docs/specifications/tests/Connectivity-TP/connectivity03.md

[ERROR]: https://github.com/zonemaster/zonemaster/blob/develop/docs/specifications/tests/SeverityLevelDefinitions.md#error
[WARNING]: https://github.com/zonemaster/zonemaster/blob/develop/docs/specifications/tests/SeverityLevelDefinitions.md#warning

--- a/docs/specifications/tests/Connectivity-TP/connectivity03.md
+++ b/docs/specifications/tests/Connectivity-TP/connectivity03.md
@@ -70,16 +70,16 @@ severity level ERROR or CRITICAL.
 
 In other cases the outcome of this Test Case is "pass".
 
-Message            |Default severity level (if message is outputted)
+Message            |Default severity level
 :------------------|:------------
-EMPTY_ASN_SET      |ERROR        
-ERROR_ASN_DATABASE |ERROR        
-IPV4_ONE_ASN       |ERROR        
-IPV4_SAME_ASN      |NOTICE       
-IPV4_DIFFERENT_ASN |INFO         
-IPV6_ONE_ASN       |ERROR        
-IPV6_SAME_ASN      |NOTICE       
-IPV6_DIFFERENT_ASN |INFO         
+EMPTY_ASN_SET      |ERROR
+ERROR_ASN_DATABASE |ERROR
+IPV4_ONE_ASN       |WARNING
+IPV4_SAME_ASN      |NOTICE
+IPV4_DIFFERENT_ASN |INFO
+IPV6_ONE_ASN       |WARNING
+IPV6_SAME_ASN      |NOTICE
+IPV6_DIFFERENT_ASN |INFO
 
 
 ## Special procedural requirements

--- a/docs/specifications/tests/Connectivity-TP/connectivity03.md
+++ b/docs/specifications/tests/Connectivity-TP/connectivity03.md
@@ -133,8 +133,9 @@ origin6.asnlookup.zonemaster.net
       *[ERROR_ASN_DATABASE]* and end these steps for Cymru look-up of 
       the specific IP address (the response was malformed).
 
-10. The ASN or ASNs from step 11 is the ASN set for that IP address
-    and is used for the further processing.
+10. Create the ASN set, for the IP address, from the ASN or ASNs from
+    the steps above and use it for the further processing.
+
 
 
 ### RIPE ASN lookup
@@ -171,8 +172,8 @@ whois -h riswhois.ripe.net " -F -M 192.0.2.10"
       *[ERROR_ASN_DATABASE]* and end these steps (the response was 
       malformed).
 
-8. From the ASN or ASNs from step 6 create the ASN set for the IP
-   address and is used for the further processing.
+8. Create the ASN set, for the IP address, from the ASN or ASNs from
+   the steps above and use it for the further processing.
 
 ## Intercase dependencies
 
@@ -182,7 +183,7 @@ None
 [Cymru database]:       #cymru-asn-lookup
 [EMPTY_ASN_SET]:        #outcomes 
 [ERROR_ASN_DATABASE]:   #outcomes 
-[IP to ASN Mapping]:    https://team-cymru.org/IP-ASN-mapping.html#dns
+[IP to ASN Mapping]:    https://team-cymru.com/community-services/ip-asn-mapping/#dns
 [IPV4_DIFFERENT_ASN]:   #outcomes 
 [IPV4_ONE_ASN]:         #outcomes 
 [IPV4_SAME_ASN]:        #outcomes 

